### PR TITLE
Added project description

### LINKED ISSUE #17: Project Descriptions
A quick description on the projects page for each project and what's included in it would be useful to those preparing to set up a contribution.
Particularly the KDE project could use this, as it's very unclear what components of the KDE project are included.

A link to the version control or primary documentation resource of each project wouldn't hurt either.

### PR DESCRIPTION
**Pull request by aminemboulouma**
Fixed #17

### CODE REVIEW

File: templates/projects.html [hunk 1]
```diff
@@ -19,6 +19,7 @@ <h4 class="project-title">{{= project.name in html}}</h4>
                 </div>
               </div>
               <div class="panel-body">
+                <p class="project-description">{{= project.description in hmtl}}</p>
```
Comment (nt1m) [inline templates/projects.html]: should be `in html`

**Comment by jankeromnes** [comment]
Thanks a lot @aminemboulouma for fixing this long-standing issue! 👍 And also it was very nice chatting with you today! @nt1m found one issue with your patch (a typo), but otherwise I don't see any problems merging this contribution. Great job! 😄

**Comment by aminemboulouma** [comment]
You are so welcome @jankeromnes, this contribution is like my first big contribution to the open source community and I really consider continuing to contribute to your project and to the community. 
I really liked chatting with you yesterday ! I hope we will catch up from time to time !

**Comment by jankeromnes** [comment]
Hi @aminemboulouma, it's so great to have you on board! Thanks for stopping by our IRC channel earlier 😄 I also really enjoyed chatting, and hope we can keep in touch!

Regarding finishing this pull request, you'll simply need to fix a small detail that @nt1m found yesterday: in the HTML template file, you accidentally wrote "in hmtl" instead of "in html". 🙂 Please add a new commit to fix this, and then we'll be able to merge your contribution, and close the issue as fixed! 👍

**Comment by aminemboulouma** [comment]
Thank you as well ! I have just done the commit to correct the typo ! Looking forward to continue working on the project !

**Comment by jankeromnes** [comment]
@aminemboulouma Congratulations on your first contribution to Janitor! 🎆 🎆 🎆 🎉 😄

**Comment by aminemboulouma** [comment]
Thank you @jankeromnes ! Yaaaaaaaaaaaaaaaaaaaay 🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉

**Pull request merged**